.github/workflows/run-tests.yml

```diff
@@ -48,3 +48,43 @@ jobs:
 
     - name: Execute Unit Tests
       run: composer test-ci
+
+  php-windows-tests:
+    runs-on: windows-2019
+    timeout-minutes: 15
+    env:
+      COMPOSER_NO_INTERACTION: 1
+
+    strategy:
+      matrix:
+        php: [8.0]
+        laravel: [8.*]
+        dependency-version: [prefer-stable]
+
+    name: P${{ matrix.php }} - L${{ matrix.laravel }} - Windows
+
+    steps:
+      - name: Set git to use LF
+        run: |
+            git config --global core.autocrlf false
+            git config --global core.eol lf
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: none
+          extensions: pdo_sqlite, fileinfo
```

Comment:
What is `fileinfo` needed for 🤔 ?

Reply:
Otherwise the packages will not install.

Reply:
Otherwise the packages will not install.